src/accordion/accordion.ts

```diff
@@ -95,9 +100,9 @@ export interface NgbPanelChangeEvent {
   template: `
   <div class="card">
     <template ngFor let-panel [ngForOf]="panels">
-      <div role="tab" id="{{panel.id}}-header" 
+      <div role="tab" id="{{panel.id}}-header" [attr.aria-selected]="panel.focused"
         [class]="'card-header ' + (panel.type ? 'card-'+panel.type: type ? 'card-'+type : '')" [class.active]="isOpen(panel.id)">
-        <a href (click)="!!toggle(panel.id)" [class.text-muted]="panel.disabled" 
+        <a href (click)="!!toggle(panel.id)" (focus)="onFocus(panel.id)" (blur)="onBlur(panel.id)" [class.text-muted]="panel.disabled" 
```

Comment:
Am I missing something, or you can do

`(focus)="panel.focused = true" (blur)="panel.focused = false"` 

directly without passing through `this._panelRefs`?